test-play-project/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.8
+sbt.version=1.3.0-RC3
```

Comment:
Is it possible to avoid using the RC version?

Reply:
No problem I can revert back to 1.2.8. (I intentionally added this version to be sure that plugin works with 1.3.0 - this version it's just at corner)